src/saurav/A24_FindVowelsCount.java

```diff
@@ -0,0 +1,32 @@
+package saurav;
+
+public class A24_FindVowelsCount {
+	
+	int cnt=0;
+	void findVowels(String input) {
+		int count = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (input.charAt(index) == 'o' || input.charAt(index) == 'a' || input.charAt(index) == 'e'
+					|| input.charAt(index) == 'i' || input.charAt(index) == 'u') {
+				count++;
+				cnt++;
```

Comment:
No usage of both the variable, you should return the count of vowels so you can use in other method for printing.